docs/documentation/writing-css.md

```diff
@@ -18,23 +18,12 @@ Every time a change happens in [application.scss](../app/assets/sass/application
 
 Try starting the app and adding some styles to `application.scss`. If you open `application.css` you should now see the compiled version of those styles.
 
-## Using the govuk_frontend_toolkit
+## Using the GOV.UK Frontend
```

Comment:
delete 'the'

Reply:
to be honest its probably best to remove this section, it's not very useful and tells people to do something thats not very clear, or we already do for them?